### ISSUE TITLE
angularRequireDependencyOrder only catching once 

### ISSUE THREAD

**Issue by kasperlewau** [tregusti/jscs-angular#18]
angularRequireDependencyOrder is only catching the first violator when using ui-router resolve blocks. 

The following structure: 

```js
$stateProvider
  .state('a', {
    resolve: {
      func: function (a, b, c, d, $http) {}
    }
  })
  .state('b', {
    resolve: {
      func: function (foo, bar, baz, $q, $stateParams) {}
    }
  })
```

Would result in jscs-angular throwing; 

> Angular dependency `$q (or $stateParams)` should be defined before baz at app/assets/boot.js : 

The `$http` violator would not be caught.

**Comment by tregusti** [comment]
Thank you, will look into it!

**Comment by tregusti** [comment]
Thank you @kasperlewau. This issue has been fixed and released as `v1.1.0`. Please report back if it works as intended.

**Comment by kasperlewau** [comment]
:heart:
